### ISSUE TITLE
Stop HTML comments showing in post previews

### ISSUE THREAD

**Issue by tog22** [tog22/eaforum#67]
When there are HTML comments `<!-- like this -->` in the first few lines of a post, they show up in the preview on the front page at http://effective-altruism.com/ . They'd need filtering out for this.

**Comment by tog22** [comment]
Would you be OK to do this @arichard4 ?

**Comment by tog22** [comment]
(No worries if not obviously! What's your general availability for tech work these days.)

**Comment by arichard4** [comment]
I start fulltime work Monday.
So not so good.

**Comment by tog22** [comment]
Gotcha! @patbl, would be great if you could try this when you have a chance!

**Comment by patbl** [comment]
Sure. I tried the following in dev and couldn't reproduce the issue:

```
<!--lol test -->
<p>rstrst</p>
<!-- like this -->
<p>huh what?</p>
```

If you have access to a post with this problem, could you send me a link, or the HTML itself (if it was your post)?

**Comment by patbl** [comment]
@tog22 Do you have an example?

**Comment by tog22** [comment]
Sure, see 'Help SHIC Get Off The Ground' at http://effective-altruism.com/user/Tom_Ash/ . By the way, I'm stepping back from .impact work so won't be dealing with its EA Forum work from now on - I've asked Georgie Mallett (the new manager of .impact) and Peter to say who the best person to tag is (it may revert to @peterhurford for a while).

**Comment by peterhurford** [comment]
Here's an example:

![](https://puu.sh/r19KL/2f4d36ba54.png)

**Comment by patbl** [comment]
Fixed by tricycle/eaforum#51.
